### ISSUE TITLE
Ternary with function call produces "Standalone expression (the line has no effect)" warning

### ISSUE THREAD

**Issue by radishes** [godotengine/godot#60410]
### Godot version

4.0 alpha 6, and my own build v4.0.alpha.custom_build [f4b0c7a1e]

### System information

Windows 10

### Issue description

Gdscript ternaries which lack assignment can produce the warning "Standalone expression (the line has no effect)." Example:

`a() if condition else b()`

I have confirmed that the correct function is being called, so the line does have an effect. The warning occurs regardless of whether `a()` or `b()` is called.

This is similar to https://github.com/godotengine/godot/issues/47088.

### Steps to reproduce

```
func a():
  print("a")

func b():
  print("b")

func _ready():
  a() if true else b()
```

**Comment by Calinou** [comment]
I can confirm this on `master` f4b0c7a1e:

![image](https://user-images.githubusercontent.com/180032/164508253-7b544de4-405e-4562-b98a-ecaa73f53bbd.png)

I can confirm this on `3.x` 3ba980379 too:

![image](https://user-images.githubusercontent.com/180032/164508081-663cd2bf-3cfe-4e71-91c7-6aef65379871.png)

**Comment by radishes** [comment]
This no longer seems to be an issue.

**Comment by akien-mga** [comment]
Seems fixed as early as 4.0-stable.
